cookbook/core/flyte_basics/decorating_workflows.py

```diff
@@ -98,7 +98,7 @@ def wrapper(*args, **kwargs):
 #
 # 1. It takes a ``before`` and ``after`` argument, both of which need to be ``@task``-decorated functions. These
 #    tasks will run before and after the main workflow function body.
-# 2. The :py:func:`~flytekit.core.node_creation.create_node` function to create nodes associated with the ``before``
+# 2. The ``create_node`` function to create nodes associated with the ``before``
```

Comment:
instead of making this a generic inline code snippet, let's figure out why `flytekit.core.node_creation.create_node` isn't being correctly linked here... it's probably because the flytekit API reference doesn't contain this function? (fixing that can be a seperate issue)

We should get into the habit of linking functions to improve discoverability of functions

Reply:
The function isn't present in API reference, hence I removed the link.

Reply:
Added GitHub link for now.